frontend/src/forms/ParcelDetailForm.test.tsx

```diff
@@ -86,103 +87,199 @@ const parcelDetailForm = (
     </Router>
   </Provider>
 );
+describe('ParcelDetailForm', () => {
+  describe('field validation', () => {
+    const exampleData = {
+      pid: '',
+      agencyId: 1,
+      address: {
+        line1: 'addressval',
+        cityId: 1,
+        provinceId: '2222',
+        postal: 'V8X3L5',
+      },
+      description: '',
+      landLegalDescription: '',
+      pin: '5',
+      zoning: 'zoningVal',
+      zoningPotential: 'zoningPotentialVal',
+      municipality: 'municipalityVal',
+      landArea: 1234,
+      statusId: 1,
+      classificationId: 1,
+      isSensitive: false,
+      latitude: 0,
+      longitude: 0,
+      evaluations: [],
+      buildings: [],
+    };
+    const fillInput = async (
```

Comment:
I like this helper and feel it could be useful in other form tests (eg ParcelFilterMap) - any chance you could extract this to a test helper class/module of it's own?

Reply:
I'd second this!  If we can make it generic enough to reuse.